### PR TITLE
feat: condition-github-actions

### PR DESCRIPTION
**Pull request by ipeshev**

### CODE REVIEW

File: analyzeCommits.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ const lastTaggedRelease = () => {
 module.exports = function (pluginConfig, config, cb) {
   // run standard commit analysis
   return analyzeCommits(pluginConfig, config, function(error, type) {
-    const branch = config.env.TRAVIS_BRANCH || config.env.GIT_LOCAL_BRANCH;
+    const branch = config.env.TRAVIS_BRANCH || config.env.GIT_LOCAL_BRANCH ||  config.env.GITHUB_REF.split('/').slice(-1)[0];
```
Comment (tsvetomir) [inline analyzeCommits.js]: Better extract this to `ghActionsBranch` function.

File: getLastRelease.js [hunk 2]
```diff
@@ -7,6 +7,8 @@ module.exports = function (pluginConfig, config, cb) {
 
   if (config.env.TRAVIS) {
     branch = config.env.TRAVIS_BRANCH;
+  } else if (config.env.GITHUB_REF) {
+    branch = config.env.GITHUB_REF.split('/').slice(-1)[0];
```
Comment (tsvetomir) [inline getLastRelease.js]: ...so we could reuse it here.

File: verifyConditions.js [hunk 3]
```diff
@@ -5,6 +5,9 @@ module.exports = function (pluginConfig, config, cb) {
   if (config.env.TRAVIS) {
     defaultVerifyConditions = require('@semantic-release/condition-travis');
     branch = config.env.TRAVIS_BRANCH;
+  } else if (config.env.GITHUB_REF) {
+    defaultVerifyConditions = require('condition-github-actions');
+    branch = config.env.GITHUB_REF.split('/').slice(-1)[0];
```
Comment (tsvetomir) [inline verifyConditions.js]: ...and here.

Comment (ipeshev) [reply verifyConditions.js]: My favourite expression :)

File: analyzeCommits.js [hunk 4]
```diff
@@ -23,7 +24,7 @@ const lastTaggedRelease = () => {
 module.exports = function (pluginConfig, config, cb) {
   // run standard commit analysis
   return analyzeCommits(pluginConfig, config, function(error, type) {
-    const branch = config.env.TRAVIS_BRANCH || config.env.GIT_LOCAL_BRANCH;
+    const branch = config.env.TRAVIS_BRANCH || config.env.GIT_LOCAL_BRANCH ||  utils.ghParseBranch(config.env.GITHUB_REF);
```
Comment (tsvetomir) [inline analyzeCommits.js]: Should pass just `config.env` and let `ghParseBranch` to figure it out.

File: utils.js [hunk 5]
```diff
@@ -0,0 +1,6 @@
+let utils = {};
+
+utils.ghParseBranch = branch => branch.split('/').slice(-1)[0];
```
Comment (tsvetomir) [inline utils.js]: `ghActionsBranch`?

**Review by tsvetomir — APPROVED** [review]

**Pull request merged**